index.php

```diff
@@ -30,21 +30,24 @@
         <?php
         if (!empty($date)) {
         ?>
-        <div class="article-date margin-bottom-tiny">
+        <div class="margin-bottom-small">
           <h3 class="font-capitalize font-bold"><?php echo $time->format('j F,'); ?><br/><?php echo $time->format('Y'); ?></h3>
         </div>
         <?php
         }
 
         if (!empty($author)) {
         ?>
-        <div class="article-author font-serif">
+        <div class="font-serif margin-bottom-small">
           <?php echo __('[:es]Por[:en]By'); ?>:<br/>
           <?php echo $author; ?>
         </div>
         <?php 
         }
         ?>
+        <div class="font-serif">
+          <a href="https://www.facebook.com/sharer/sharer.php?u=<?php the_permalink(); ?>" target="_blank" rel="noopener noreferrer"><?php _e('[:en]Share[:es]Compartir'); ?><br><?php _e('[:en]on[:es]en'); ?> Facebook</a>
```

Comment:
possible the `<a>` should wrap the `<div>` so the whole thing is clickable not just the text (on the linebreak the space between isn't a link)